api/tests/organizational-entities/acceptance/application/certification-center/certification-center.admin.route.test.js

```diff
@@ -380,4 +381,47 @@ describe('Acceptance | Organization Entities | Admin | Route | Certification Cen
       });
     });
   });
+
+  describe('POST /api/admin/certification-centers/{certificationCenterId}/archive', function () {
+    it('returns 204 if certification center has been archived', async function () {
```

Comment:
Un `if` ainsi dans la description du test rend la lecture plus difficile. Alors voici une suggestion, mais il y a peut-être mieux : 
```suggestion
    it('archives the certification center and related data and returns 204', async function () {
```